pkg/chains/storage/gcs/gcs.go

```diff
@@ -185,41 +240,68 @@ type TaskRunStorer struct {
 	key string
 }
 
-// Store stores the
+// Store stores the TaskRun chains information in GCS
+//
+//nolint:staticcheck
 func (s *TaskRunStorer) Store(ctx context.Context, req *api.StoreRequest[*v1beta1.TaskRun, *in_toto.Statement]) (*api.StoreResponse, error) {
+	tr := req.Artifact
+	return store(ctx, s.writer, "taskrun", s.key,
+		tr.GetNamespace(), tr.GetName(), string(tr.GetUID()),
+		req.Bundle.Signature, req.Bundle.Content, req.Bundle.Cert, req.Bundle.Chain)
+}
+
+// PipelineRunStorer stores PipelineRuns in GCS.
+type PipelineRunStorer struct {
+	writer gcsWriter
+
+	// Optional key to store objects as. If not set, the object UID will be used.
+	// The resulting name will look like: $bucket/pipelinerun-$namespace-$name/$key.signature
+	key string
+}
+
+// Store stores the PipelineRun chains information in GCS
+//
+//nolint:staticcheck
+func (s *PipelineRunStorer) Store(ctx context.Context, req *api.StoreRequest[*v1beta1.PipelineRun, *in_toto.Statement]) (*api.StoreResponse, error) {
+	pr := req.Artifact
+	return store(ctx, s.writer, "pipelinerun", s.key,
+		pr.GetNamespace(), pr.GetName(), string(pr.GetUID()),
+		req.Bundle.Signature, req.Bundle.Content, req.Bundle.Cert, req.Bundle.Chain)
+}
+
+func store(ctx context.Context, writer gcsWriter, kind, key, namespace, name, uid string,
```

Comment:
This is quite a bit of params, and of the same type - easy to make a mistake.

Would it make sense to combine `kind, key, namespace, name, uid` into a single `prefix` parameter?

Reply:
Done, changed the params to use a `prefix` param directly instead of the additional 4 params mentioned - `kind, key, namespace, name, uid